types/react-animals/tsconfig.json

```diff
@@ -11,7 +11,12 @@
         "jsx": "preserve",
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "paths": {
+            "react": [
+                "../../react/v16/index.d.ts"
```

Comment:
Matching runtime peer dependencies: https://github.com/arvinpoddar/react-animals/blob/1fe6b0c21682f999d6f8d6092e55e5144f09cf0e/package.json#L13

Reply:
can this be done via `package.json` deps? like in `enzyme`?
```json
"@types/react": "^16"
```
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/enzyme/package.json#L10